src/macaron/artifact/maven.py

```diff
@@ -140,3 +140,10 @@ def create_maven_purl_from_artifact_filename(
             )
 
     return None
+
+
+def is_valid_maven_group_id(group_id: str) -> bool:
+    """Check if the provided string is a valid maven group id."""
```

Comment:
Please add `Parameters` and `Returns` sections to the docstrings. Please have a look at the other docstrings in the codebase.

Reply:
For more information on our doc strings guide, you can have a look at: https://github.com/oracle/macaron/blob/main/docs/source/pages/developers_guide/style_guide.rst#docstrings

Reply:
I had forgotten about the guide, thanks :slightly_smiling_face: